CHANGELOG.md

```diff
@@ -17,6 +16,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    this feature was anonymously sponsored: thank you to our sponsor!
     - Add `legend.xref` and `legend.yref` to enable container-referenced positioning of legends [[#6589](https://github.com/plotly/plotly.js/pull/6589)], with thanks to [Gamma Technologies](https://www.gtisoft.com/) for sponsoring the related development.
     - Add `colorbar.xref` and `colorbar.yref` to enable container-referenced positioning of colorbars [[#6593](https://github.com/plotly/plotly.js/pull/6593)], with thanks to [Gamma Technologies](https://www.gtisoft.com/) for sponsoring the related development.
+  - `px` methods now accept data-frame-like objects that support a `to_pandas()` method, such as polars, cudf, vaex etc
```

Comment:
@nicolaskruchten Looks like `vaex` doesn't have `to_pandas` method, only [`to_pandas_df`](https://vaex.io/docs/_modules/vaex/dataframe.html#DataFrame.to_pandas_df).

Therefore, tests with vaex do not pass in https://github.com/plotly/plotly.py/pull/4244.